generated/proto/policy.proto

```diff
@@ -34,3 +34,12 @@ message Policy {
 	Resolution resolution = 1;
 	Retention retention = 2;
 }
+
+message CompressedPolicy {
+	Policy policy = 1;
+	int64 id = 2;
+}
+
+message ActivePolicies {
```

Comment:
I don't think the cutover based lookup will be that expensive, since most of the lookup results should be cached anyways. That said, since we can guarantee the ids for existing policies should never change, we could probably just store the latest snapshot here alongside the cutover time (that takes into account kv propagation delay), so when the collector gets the update, it simply swaps the old existing policy list with the new list, so we avoid the cutover time lookup overhead.